.github/workflows/release-package.yml

```
@@ -0,0 +1,36 @@
+name: Release Package
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+'
```
I configured the tag rule to expect a `v` prefix. Please let me know if this is an issue and I can update the rule.